pulser/register.py

```diff
@@ -282,6 +282,29 @@ def _draw_checks(
                     "Needs more than one atom to draw " "the blockade radius."
                 )
 
+    @abstractmethod
```

Comment:
What's the motivation to define it as an `abstractmethod` that is the same on all subclasses? Is it to force the definition of `_to_dict()` on each subclass (even if redundant) so it requires coverage by the unit tests?

Reply:
The motivation is to make sure that any future subclass won't be wrongly serializable by default, and will need to make it explicit by adding a new `_to_dict()` method. The developer will also be forced to think about whether they want the method to be identical, or whether they want to extend it.

Reply:
Yes, that's a fair point.